src/lib/gssapi/krb5/accept_sec_context.c

```diff
@@ -706,8 +706,12 @@ kg_accept_krb5(minor_status, context_handle,
             goto fail;
         }
 
-        /* Use ap_options from the request to guess the mutual flag. */
-        gss_flags = GSS_C_REPLAY_FLAG | GSS_C_SEQUENCE_FLAG;
+        /*
+         * Use ap_options from the request to guess the mutual flag.
+         * In the case of a gss_set_context_flags() call, make sure to
```

Comment:
I don't think that's quite right.  We want to have a distinct field in the context for "requested flags" and "flags we actually got".